apply/default_applier.go

```diff
@@ -74,6 +75,9 @@ var ActionFuncMap = map[ActionName]func(*DefaultApplier) error{
 	Guest: func(applier *DefaultApplier) error {
 		return applier.Guest.Apply(applier.ClusterDesired)
 	},
+	CNI: func(applier *DefaultApplier) error {
```

Comment:
I think applier not care about CNI in the feature

Reply:
so , we not support only change cni configure ?